.env.example

```diff
@@ -49,6 +49,7 @@ TWILIO_MESSAGE_VALIDITY_PERIOD=
 DST_REFERENCE_TIMEZONE='US/Eastern'
 PASSPORT_STRATEGY=local
 TEXTER_SIDEBOXES=celebration-gif,default-dynamicassignment,default-releasecontacts,contact-reference,tag-contact,freshworks-widget,default-editinitial,take-conversations,hide-media,texter-feedback,contact-notes
+DYNAMICASSIGNMENT_BATCHES=finished-replies-tz,vetted-texters,finished-replies
```

Comment:
I believe we'll need to update the docs for this: docs/REFERENCE-environment_variables.md

Reply:
Good catch. This change might not be entirely necessary. It is only when trying to get "vetted-texterassignment" to work. So if you were to just do `DYNAMICASSIGNMENT_BATCHES=vetted-texterassignment", that entire feature would brick as it would need those other three.